### ISSUE TITLE
VPN state

### ISSUE THREAD

**Issue by andrey-pr** [python-sdbus/python-sdbus-networkmanager#59]
How to get current VPN connection status?
I tried
```python
conn = ActiveVPNConnection(connection_path)
print(conn.vpn_state)
```
but got `<sdbus.dbus_proxy_async_property.DbusPropertyAsyncBinded object at 0x7f448eccf0d0>`.
How to get connection status correctly?

**Comment by igo95862** [comment]
If you are using async you need to `print(await conn.vpn_state)`

**Comment by andrey-pr** [comment]
I got `sdbus.dbus_exceptions.DbusInvalidArgsError: Interface «org.freedesktop.NetworkManager.VPN.Connection» not exist`

**Comment by igo95862** [comment]
What is your `connection_path`?

Is it in the form of `/org/freedesktop/NetworkManager/ActiveConnection/X`?

**Comment by igo95862** [comment]
> Thank you. I tried `/org/freedesktop/NetworkManager/Settings/22`, but actual is `/org/freedesktop/NetworkManager/ActiveConnection/16`.
> 
> Now it returns `5`. Where can I see table of meaning of this integer?

[`VpnState enum`](https://github.com/python-sdbus/python-sdbus-networkmanager/blob/76a3c2cc4c7ac07fa220637f89eab2f776daab96/sdbus_async/networkmanager/enums.py#L704)

**Comment by igo95862** [comment]
Hmm... Looks like the enum values have changed? Upstream documentation says that 5 is "The VPN connection is active."

https://networkmanager.dev/docs/api/latest/nm-vpn-dbus-types.html#NMVpnConnectionState

**Comment by igo95862** [comment]
Looks like the current enum `VpnState` actually refers to `NMVpnServiceState`.

**Comment by igo95862** [comment]
I added [`VpnConnectionState`](https://python-sdbus-networkmanager.readthedocs.io/en/latest/enums.html#sdbus_async.networkmanager.enums.VpnConnectionState) and renamed `VpnState` to `VpnServiceState`.

This will be available in version 3.0.
